lucid/launcher.py

```diff
@@ -78,6 +78,12 @@ def get_parser():
         action='store_true',
         default=False
     )
+    parser.add_argument(
+        '--skip_happi',
```

Comment:
Consistency-wise this argument makes sense (above arguments are in the style `--a_b`)
In the future, it'd be nice to switch to `--a-b` as it's more ergonomic from a CLI user standpoint

Reply:
I had the same thought, but wasn't sure if it was just a flight of whimsy on my end. 👍